README.md

```diff
@@ -67,6 +67,7 @@ If you want to support IE9, you will need to use a classList pollyfill, like Eli
 * __forcePlainText__: Forces pasting as plain text. Default: true
 * __placeholder__: Defines the default placeholder for empty contenteditables when __disablePlaceholders__ is not set to true. You can overwrite it by setting a data-placeholder attribute on your elements. Default: 'Type your text'
 * __secondHeader__: HTML tag to be used as second header. Default: h4
+* __standardizeSelectionStart__: Standardizes how the beginning of a range is decided between browsers whenever the selected text is analyzed for updating toolbar buttons status
```

Comment:
Looking for any input here.  This will definitely fix a bug in Firefox, and I tried to test it as thoroughly as I could with other browsers. I just worry about other edge cases, so I figured putting this behind an option that defaults to false would be safer.

However, this is a bug fix, so we might want to just have this around always, or at least defaulted to true....anyone have any thoughts?